tests/testthat/test-renderer1-global-variables.R

```diff
@@ -14,7 +14,7 @@ myArray;'
 
 getVariables <- function(){
   
-  vars<-runtime_evaluate(script=myScript, return.value=TRUE)
+  vars<-runtime_evaluate(script=myScript)
```

Comment:
please use space before and after <- please use find/replace or sed or grep to fix all of these